content/plugin/framework/resources/index.mdx

```diff
@@ -162,5 +164,6 @@ func (p *provider) GetResources(_ context.Context) (map[string]tfsdk.ResourceTyp
 
 ## Further Resource Capabilities
 
+- [Import](/plugin/framework/resources/import) helps practitioners bring existing infrastructure under Terraform's management.
```

Comment:
also one thought I had here - do we need to specify that import is required whereas the others are optional? It feels a bit odd to list that here when my impression is that this is absolutely required, whereas the others are just extra stuff you can add on (right?). Maybe we should take it out of this section and just link to the examples subpage above in the `ImportState` method section....thoughts?

```suggestion
- [Import](/plugin/framework/resources/import) lets Terraform begin managing existing infrastructure resources. Each resource's `ImportState` method must either enable the `Read` method to refresh [`tfsdk.Resource`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/tfsdk#Resource) or return an error.
```

Reply:
Let's remove this then!

```suggestion
```